### PR TITLE
[Android] Fixed SwipeItems size dynamically changing the Content Margin

### LINKED ISSUE #4121: [Android] Dynamically changing SwipeView margins is not updating the SwipeItems
### Description

Dynamically changing SwipeView margins is not updating the SwipeItems
image

### Steps to Reproduce

Use .NET MAUI Gallery SwipeView margin sample to validate the issue.

### Version with bug

Preview 13

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Fixed issue setting SwipeItems size by dynamically changing the Margin/Padding of the content.
fix-swipeview-margin

To validate the changes, launch the .NET MAUI Gallery on Android and navigate to the SwipeView samples. Select the "SwipeView Margin" sample and change the margin content. Then, try to open the SwipeView. If the SwipeItem is rendered correctly (centered text), the test has passed.

### Issues Fixed

Fixes #4121

### CODE REVIEW

File: src/Controls/src/Core/HandlerImpl/SwipeView.Impl.cs [hunk 1]
```diff
@@ -112,6 +114,15 @@ private protected override void OnParentChangedCore()
 			}
 		}
 
+		void UpdateMargin()
+		{
+			if (this is not ISwipeView swipeView)
```
Comment (mandel-macaque) [inline src/Controls/src/Core/HandlerImpl/SwipeView.Impl.cs:119]: <3

File: src/Core/src/Handlers/SwipeItemMenuItem/SwipeItemMenuItemHandler.Android.cs [hunk 2]
```diff
@@ -110,14 +111,23 @@ int GetIconSize()
 
 		void UpdateSize()
 		{
-			var textSize = 0;
-			var contentHeight = 0;
-
 			var mauiSwipeView = PlatformView.Parent.GetParentOfType<MauiSwipeView>();
+
```
Comment (mandel-macaque) [inline src/Core/src/Handlers/SwipeItemMenuItem/SwipeItemMenuItemHandler.Android.cs:115]: Reduce the diff ;)

**Review by mandel-macaque — APPROVED** [review]

**Comment by rmarinho** [comment]
I was testing this with iOS also and saw that seems Android doesn't clip to the container when we swipe. Not sure if related with this issue. Also not sure what's the correct one , iOS or Android.
Screenshot 2023-03-20 at 10 29 24

**Review by rmarinho — APPROVED** [review]
The clip issue doesn't seem related with this PR so I m approving. But can you take a look at that issue @jsuarezruiz  because in your gif it does seem to clip so I guess this might be a regression somewhere?

**Comment by jsuarezruiz** [comment]
> The clip issue doesn't seem related with this PR so I m approving. But can you take a look at that issue @jsuarezruiz because in your gif it does seem to clip so I guess this might be a regression somewhere?

Thanks for your review Rui, and for notice the clip issue. The clip issue will be fixed by https://github.com/dotnet/maui/pull/14024

**Review by rmarinho — APPROVED** [review]

**Pull request merged**